### ISSUE TITLE
For Thong

### ISSUE THREAD

**Issue by milailov2021** [milailov2021/chat#4]
Hi, frd.
Nice to meet you here.

**Comment by milailov2021** [comment]
Are you there?

**Comment by thongtran8197** [comment]
yes

**Comment by thongtran8197** [comment]
how are you?

**Comment by milailov2021** [comment]
I'm so glad to meet you here. I'm sorry for this uncomfortable communication, but hope your understanding.

**Comment by milailov2021** [comment]
Are you there?

**Comment by thongtran8197** [comment]
I am here.

**Comment by milailov2021** [comment]
Really good.

**Comment by milailov2021** [comment]
I have earned some money, please withdraw it.

**Comment by thongtran8197** [comment]
How about your work, bro?

**Comment by milailov2021** [comment]
good, I can pay you money.

**Comment by thongtran8197** [comment]
> I have earned some money, please withdraw it.

How many i will withdraw?

**Comment by milailov2021** [comment]
please 700usd.
200usd is yours.

**Comment by thongtran8197** [comment]
Got it.

**Comment by milailov2021** [comment]
when will you?

**Comment by milailov2021** [comment]
please 350usd to  one of those:
USDT(BEP20, ERC20): 0x0ED04a31bc3CD1B0ef26bD64fB48D1acd497Aa9A
USDT(TRC20): TH8irg12XkocQ6Z3T1vCH2eVJUyznPKzhg

**Comment by thongtran8197** [comment]
Tomorrow!! Will ping you log out before I withdraw

**Comment by milailov2021** [comment]
please transfer 350usd to the above account.(please save account info, and then I will remove it.)
and transfer 150usd to another account(I will send you tomorrow.)

**Comment by milailov2021** [comment]
okay?

**Comment by thongtran8197** [comment]
> please 350usd to one of those: USDT(BEP20, ERC20): 0x0ED04a31bc3CD1B0ef26bD64fB48D1acd497Aa9A USDT(TRC20): TH8irg12XkocQ6Z3T1vCH2eVJUyznPKzhg

And 150usd? Because I will send you 500 usd

**Comment by thongtran8197** [comment]
> okay?

Okay

**Comment by milailov2021** [comment]
I will send the account info to receive 150usd in tomorrow.
Have you saved the above account info?

**Comment by thongtran8197** [comment]
Yes, let delete the message

**Comment by milailov2021** [comment]
okay.

**Comment by milailov2021** [comment]
please delete messages on your side.

**Comment by thongtran8197** [comment]
Bye. Let continue your working.

**Comment by thongtran8197** [comment]
Let reload the page

**Comment by milailov2021** [comment]
How's your work?

**Comment by thongtran8197** [comment]
Ah, I have more free time on next months. And I am investigating Freelancer. I want to ask you about different between membership 9usd/month and 49usd/month (I know basic is 100 with 300 bid/month).

**Comment by milailov2021** [comment]
yup, 
When you buy 9usd membership, you have 100bids and it will add 100bids per month.
When you buy 49usd membership, then you have 300bids and it will add 300bids per month.
In addition to that, you can add your skills up to 80 for 9usd per month, while you can add up to 100 for 49usd per month.

**Comment by milailov2021** [comment]
Then are you going to use your freelancer account?

**Comment by thongtran8197** [comment]
No, i use my friend accounts. I have multiple verified account bro.

**Comment by milailov2021** [comment]
okay, how are you spending your weekend?

**Comment by milailov2021** [comment]
what's happened on your company?

**Comment by thongtran8197** [comment]
Watch football bro

**Comment by thongtran8197** [comment]
> what's happened on your company?

Normal as all days keke

**Comment by milailov2021** [comment]
hehe

**Comment by milailov2021** [comment]
have a good night,
let's meet tomorrow. Will contact you.

**Comment by thongtran8197** [comment]
Okay

**Comment by thongtran8197** [comment]
i am requesting withdraw.

**Comment by milailov2021** [comment]
i have just checked. but I was working. LOL
it will go well.

**Comment by thongtran8197** [comment]
yep, i requested by mobile app. hope it go well.

**Comment by milailov2021** [comment]
how's your today?

**Comment by thongtran8197** [comment]
take a coffee with family. Today is Sunday haha.

**Comment by milailov2021** [comment]
How nice day. pleasure.

**Comment by milailov2021** [comment]
Have a good time.

**Comment by thongtran8197** [comment]
thank you. You wake up so soon. I see 6:30 AM.

**Comment by milailov2021** [comment]
I works so hardly. Hahaha

**Comment by thongtran8197** [comment]
The withdraw has issue due to your ptoject. Do you know any reason?

**Comment by milailov2021** [comment]
Freelancer platform is really dirty one.

**Comment by thongtran8197** [comment]
Hi

**Comment by milailov2021** [comment]
Hi

**Comment by milailov2021** [comment]
What are they saying?

**Comment by thongtran8197** [comment]
Before mailing support, i want to know total project fee acceptance is 57,5 usd. Right?

**Comment by milailov2021** [comment]
yes, hold on i will share the screenshot.

**Comment by milailov2021** [comment]
![image](https://github.com/user-attachments/assets/e4b42e99-f252-458d-bfcf-8d64f117f1e6)

**Comment by milailov2021** [comment]
total 57.5usd

**Comment by thongtran8197** [comment]
From supporter:
Upon checking, your client on the Project:Network Security Enhancement with RSA Encryption has an account issues that they need to resolved first. 

13:57
No worries, I can send an email to your client to assist them in resolving their account issues.

**Comment by thongtran8197** [comment]
We have to waiting new from the client

**Comment by milailov2021** [comment]
thanks for your help. I'm really worried.

**Comment by thongtran8197** [comment]
yep, i will continue tracking the progress.

**Comment by milailov2021** [comment]
Bro, please withdraw 125usd firstly, and take it.
